Doc/library/tkinter.rst

```diff
@@ -95,6 +95,568 @@ Or, more often::
 
    .. FIXME: The following keyword arguments are currently recognized:
 
+   Methods inheritted from :class:`Misc` parent class (some methods ommited):
```

Comment:
```suggestion
   Methods inherited from :class:`!Misc` parent class (some methods omited):
```
Made spelling fixes; Added ! to Misc since there is not such doc section.